pallets/settlement/src/lib.rs

```diff
@@ -781,13 +788,20 @@ decl_module! {
             Self::deposit_event(RawEvent::VenuesBlocked(did, ticker, venues));
             Ok(())
         }
+
+        /// An internal call to execute a scheduled settlement instruction.
+        #[weight = 500_000_000]
+        fn execute_scheduled_instruction(origin, instruction_id: u64) -> DispatchResultWithPostInfo {
```

Comment:
For my education, can the scheduler pallet call non-dispatchables?

Reply:
It should be a `call: <T as pallet_scheduler::Trait>::Call` which I believe is always a dispatchable.